_posts/2018-02-28-foreman-community-newsletter---february-2018.md

```diff
@@ -0,0 +1,120 @@
+testing and fixes! Please do continue to log the [bugs][issues] you
+find in order to make this release better.
+
+### Katello 3.5.1 released
+
+Katello 3.5.1 is here! This version works with Foreman 1.16 and provides
+features and bug fixes all around - installer, content views, API and more.
+More details in the [announcement][k_3_5_1]. Please do [log][k_issues] any issues
+you find with this.
+
+
+### Katello 3.6 RC1 out
+
+Katello 3.6 is available for testing! This version will work with Foreman 1.17.
+This brings in a lot of features and bug fixes which can be seen in the
+[change log][k_3_6_rc1]. Reporting of [bugs][k_issues] will be appreciated.
+
+### Community survey 2018
+
+The survey is still open. We have some nice responses and we want more!
+Let's beat last years responses and help us shape the direction of the
+project.
+
+http://goo.gl/RNY8MY
+
+### Upcoming Events
+
+* Community Demo - 8th March (URL to be announced)
+
+Time again for the latest round-up of action from the community, covering core,
+proxies, plugins, and more.  As always, you're welcome to participate in the
+show, either via YouTube Live chat, or via our IRC channel
+(Freenode/#theforeman).
+
+* [Deep Dive - Foreman-Probing - 7th March](https://www.youtube.com/watch?v=-PYhFH7YraU)
+In this deep dive we'll meet a new plugin, foreman-probing.
+This plugin allows scanning networks and creating Hosts in Foreman for
+the detected machines. Adam will describe the plugin's features, inner
+working and possible usage for brownfield deployments.
+
+* [Deep Dive - Telemetry in Foreman - 14th March](https://www.youtube.com/watch?v=QoJ-r8YfWEI)
+Rails internal telemetry was recently added into Foreman Core, in this demo
+Lukáš Zapletal is going to configure and show how to integrate Foreman with Prometheus.
+The next part will cover all currently exported metrics, how to read them and utilize
+for finding bottlenecks. In the last part, he is going to show how to add your own
+telemetry data into core or plugins.
+
+#### Community Demos / Deep Dives
+
+* [UX review - Red Hat Repositories page - 12th Feb](https://www.youtube.com/watch?v=9o1X_Th5Tus)
+This demo provided an in-depth look at the new Red Hat Repositories page.
+A page that has been rewritten using React. It adds search and filtering capabilities
+and brings that page inline with patternfly best practices.
+
+* [Community Demo - 15th Feb](https://www.youtube.com/watch?v=romjVfktj6E)
+This time we shared experiences from the latest conferences, 
+showed our ansible integration updates, hammer improvements, 
+changes and new features in remote execution, reports and searchable fields in the API.
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+### Blog posts
+
+##### Running Foreman with PostgreSQL 9.6 from Software Collections
+When deploying Foreman on CentOS 7, by default you get PostgreSQL 9.2
+which is officially EOL upstream.
```

Comment:
I'd mention that 9.6 should perform faster, but AFAIK some katello tools don't support this. Foreman doesn't provide those tools so it doesn't care. @evgeni?

Reply:
Yeah, @katello needs a version of `postgresql-jdbc` that is compatible with 9.6, and the one in CentOS is not, so Candlepin fails. Otherwise it would work (after we clean up the RPM deps). That said, the post only talks about @theforeman ;)